src/chrome/content/rules/Steam.xml

```diff
@@ -1,53 +1,55 @@
 
 	<target host="steamcommunity.com" />
-		<test url="http://steamcommunity.com/" />
-		<test url="http://steamcommunity.com/broadcast/getbroadcastmpd/?steamid=76561197966726415" />
-	<target host="*.steamcommunity.com" />
-		<test url="http://www.steamcommunity.com/" />
-		<test url="http://api.steamcommunity.com/" />
-		<test url="http://cdn.steamcommunity.com/public/css/skin_1/header.css" />
+	<target host="www.steamcommunity.com" />
+	<target host="api.steamcommunity.com" />
+	<target host="cdn.steamcommunity.com" />
 
+	<target host="steamgames.com" />
+	<target host="www.steamgames.com" />
 	<target host="partner.steamgames.com" />
 
+	<target host="steampowered.com" />
+	<target host="www.steampowered.com" />
 	<target host="api.steampowered.com" />
-		<test url="http://api.steampowered.com/ISteamWebAPIUtil/GetSupportedAPIList/v1/" />
-	<target host="kgs.steampowered.com" />
+	<target host="cdn.steampowered.com" />
+	<target host="help.steampowered.com" />
+	<target host="media.steampowered.com" />
 	<target host="partner.steampowered.com" />
-	<target host="storefront.steampowered.com" />
-		<test url="http://storefront.steampowered.com/app/440/" />
 	<target host="store.steampowered.com" />
-		<test url="http://store.steampowered.com/app/440/" />
-	<target host="support.steampowered.com" />
-	<target host="help.steampowered.com" />
-	<target host="cdn.steampowered.com" />
-		<test url="http://cdn.steampowered.com/public/images/gift/steamcards_promo_02.png" />
 	<target host="cdn.store.steampowered.com" />
-		<test url="http://cdn.store.steampowered.com/public/images/gift/steamcards_promo_02.png" />
-	<target host="www.steampowered.com" />
+	<target host="storefront.steampowered.com" />
+	<target host="support.steampowered.com" />
 
-	<securecookie host="^steamcommunity\.com$" name=".+" />
-	<securecookie host="^partner\.steamgames\.com$" name=".+" />
-	<securecookie host="^partner\.steampowered\.com$" name=".+" />
+	<target host="cdn.akamai.steamstatic.com" />
+	<target host="cdn.edgecast.steamstatic.com" />
+	<target host="community.akamai.steamstatic.com" />
+	<target host="community.edgecast.steamstatic.com" />
+	<target host="store.akamai.steamstatic.com" />
+	<target host="store.edgecast.steamstatic.com" />
 
-	<rule from="^https?://cdn\.steamcommunity\.com/"
-			to="https://steamcommunity-a.akamaihd.net/" />
+	<securecookie host=".+" name=".+" />
 
-	<rule from="^https?://storefront\.steampowered\.com/"
+	<rule from="^http://(www\.)?steamgames\.com/"
 			to="https://store.steampowered.com/" />
 
-	<rule from="^https?://((cdn|cdn\.store|www)\.)?steampowered\.com/"
+	<rule from="^http://((cdn|storefront|www)\.)?steampowered\.com/"
 			to="https://store.steampowered.com/" />
 
-	<rule from="^https?://www\.steamcommunity\.com/"
+	<rule from="^http://www\.steamcommunity\.com/"
 			to="https://steamcommunity.com/" />
 
+	<rule from="^http://(cdn\.steamcommunity|community\.akamai\.steamstatic|community\.edgecast\.steamstatic)\.com/"
```

Comment:
This rule does not work for deep link

https://steamcommunity.com/economy/image/XthyL4-kdnXfOarfbSh-4uaUuJ7L1Hg1C_d5xYnONkWX00XU1rgwZ3_gtiFLSCJs_pPj2cLLeDEN_H7XiNghT5iRRs_AqTdxRKe2IUgUbHHvkLiBmp45Bw_0dcrExGxGkowMjMS5dTYu_OAtFF51fe-F_4vEyzRqDKtyx8HYdkSQ31Ddlu1zLGugsg==/99fx66f

Reply:
This rule doesn't cover the URL you posted?

Reply:
You are right. Sorry about that:

http://cdn.steamcommunity.com/economy/image/XthyL4-kdnXfOarfbSh-4uaUuJ7L1Hg1C_d5xYnONkWX00XU1rgwZ3_gtiFLSCJs_pPj2cLLeDEN_H7XiNghT5iRRs_AqTdxRKe2IUgUbHHvkLiBmp45Bw_0dcrExGxGkowMjMS5dTYu_OAtFF51fe-F_4vEyzRqDKtyx8HYdkSQ31Ddlu1zLGugsg==/99fx66f

Reply:
What's the problem? If you change the domain to akamai cdn, it still works.

https://steamcommunity-a.akamaihd.net/economy/image/XthyL4-kdnXfOarfbSh-4uaUuJ7L1Hg1C_d5xYnONkWX00XU1rgwZ3_gtiFLSCJs_pPj2cLLeDEN_H7XiNghT5iRRs_AqTdxRKe2IUgUbHHvkLiBmp45Bw_0dcrExGxGkowMjMS5dTYu_OAtFF51fe-F_4vEyzRqDKtyx8HYdkSQ31Ddlu1zLGugsg==/99fx66f

Reply:
We usually try to avoid this because it makes rulesets fragile. As soon as they change their CDN provider, it breaks things.

If @apaloma can get someone from Valve to *officially* comment that they reviewed the complex, I am okay with doing an exception.

Reply:
I've pinged him about this. But subdomains `cdn.steampowered.com`, `cdn.store.steampowered.com` and `cdn.steamcommunity.com` aren't really used. They use `akamai.steamstatic.com` and `akamaihd.com` variants on store and community.

Reply:
Yes, @Bisaloo I work for Valve (http://www.valvesoftware.com/email.php?recipient=Tony+Paloma) and provided @xPaw with rules here: https://github.com/EFForg/https-everywhere/pull/15087#issuecomment-379330255

It is possible we change CDN at some point in the future, but unlikely that occurs before we're 100% HTTPS.